src/Components/Milestones/Assessment.js

```diff
@@ -1,79 +1,45 @@
 import React, { useEffect, useState } from 'react';
 import './Assessment.css';
-import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
 import MilestoneClientInfo from './MilestoneClientInfo';
 import MilestoneChecklist from './MilestoneChecklist';
-import { fetchJobseekerData } from '../../Services/jobseeker-data-service';
-import {
-  objToArray, skillsChecklistOptions, industryInterestOptions,
-} from '../../Services/objects-service';
 
-const mockData = {
-  'Authentic Name': 'john smith',
-  Pronouns: 'they/them',
-  Phone: '(408) 263-0181',
-  Email: 'johnsmith@gmail.com',
-  Location: 'los angeles, ca',
-  Ethnicity: 'new mexican',
-  Age: '29',
-  'Gender Identity': 'nonbinary',
-  Sexuality: 'bisexual',
-  Veteran: 'no',
-  Disability: 'no',
-  Housing: 'apartment',
-  'Currently Employed': 'yes',
-  'Prior Convictions': 'none',
-};
+function mapArrays(degreeArray, string) {
```

Comment:
I think the map -> array fx is fixed if you want to just import that

Reply:
I tried using objToArray...is that the right one? If it is, there were some issues :o so idk if we can just leave the code to how it is?